content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
@brian-brazil The various go metric abstractions are essentially wrappers, whether they claim so or not.

In any case, since @tomcz created this, being familiar with both Clojure and Prometheus, I'd assume that many other Clojure programmers will feel the same urge when they use the naked Java client. Rather than letting them all create utility functions (with likely the same or even more errors), wouldn't it be better if we point them to a repo where they can join forces and even enjoy our consulting on best practices?

Speaking of which, perhaps you could file issues in https://github.com/soundcloud/prometheus-clj to help @tomcz getting naming and labeling in line with best practices?